app/models/server/models/LivechatRooms.js

```diff
@@ -578,6 +578,31 @@ export class LivechatRooms extends Base {
 
 		return this.update(query, update);
 	}
+
```

Comment:
This implementation needs to be implemented on the `ee` directory Model.
We have an  `Enterprise LivechatRooms` model.